app/kumactl/cmd/install/testdata/install-transparent-proxy.defaults.golden.txt

```diff
@@ -8,12 +8,12 @@
 -A PREROUTING -p tcp -j KUMA_MESH_INBOUND
 -A OUTPUT -p tcp -j KUMA_MESH_OUTBOUND
 -A KUMA_MESH_INBOUND -p tcp -j KUMA_MESH_INBOUND_REDIRECT
--A KUMA_MESH_OUTBOUND -s subnetPlaceholder/mask -o ifPlaceholder -j RETURN
--A KUMA_MESH_OUTBOUND -p tcp -o ifPlaceholder ! -d subnetPlaceholder/mask -m owner --uid-owner 0 -j KUMA_MESH_INBOUND_REDIRECT
+-A KUMA_MESH_OUTBOUND -s 127.0.0.6/32 -o ifPlaceholder -j RETURN
+-A KUMA_MESH_OUTBOUND -p tcp -o ifPlaceholder ! -d 127.0.0.1/32 -m owner --uid-owner 0 -j KUMA_MESH_INBOUND_REDIRECT
```

Comment:
AFAIK the subnetPlaceholder was there to avoid issues when testing on ipv6 machines

Reply:
That's correct, but it was only happening because there was a bug when installing in dry run mode which was generating only one version of rules (usually IPv6). It was fixed in: https://github.com/kumahq/kuma/pull/10619